modules/nlu/src/backend/storage.ts

```diff
@@ -123,13 +124,8 @@ export default class Storage {
   }
 
   async getIntents(): Promise<sdk.NLU.IntentDefinition[]> {
-    const debug = DEBUG('nlu', this.botId)
     const intentsName = await this.botGhost.directoryListing(this.intentsDir, '*.json')
-    const intents = await Promise.mapSeries(intentsName, name =>
-      this.getIntent(name).catch(err => {
-        debug(`An error occured while loading intents: ${err}`)
-      })
-    )
+    const intents = await Promise.mapSeries(intentsName, name => this.getIntent(name).catch(x => undefined))
```

Comment:
That catch will swallow the 2 other exceptions throws in the getIntent method. I think it would make more sense to throw exceptions in the getIntent and use a logger in the catch to warn